src/main/java/com/samoilenko/onlinebookstore/dto/CreateBookRequestDto.java

```diff
@@ -0,0 +1,13 @@
+package com.samoilenko.onlinebookstore.dto;
+
+import java.math.BigDecimal;
+
+public class CreateBookRequestDto {
+    private Long id;
```

Comment:
Request DTO should be without id. I don't think that assigning ids when creating a book is a good idea.

Reply:
agree